### ISSUE TITLE
Add support for teams Dropbox account

### ISSUE THREAD

**Issue by v-aravind** [files-community/Files#3083]
**Describe the bug**
I have two dropbox accounts, a personal and a work one. Only my personal account shows up under 'drives', not the work one. I also have two Onedrive accounts, but these both show up properly. So this seems to be a Dropbox specific issue.

**Desktop (please complete the following information):**
 - OS Version: Windows 10 Enterprise 20H2
 - App version: v1.1

**Comment by gave92** [comment]
Thanks for this report. Could you post here the contents of this file? `%LocalAppData%\Dropbox\info.json`
It should look something like:
```json
{"personal": {"path": "SOME_PATH", "host": XXXXX, "is_team": false, "subscription_type": "Basic"}}
```
Before uploading remove personal info (`path` and `host`

**Comment by v-aravind** [comment]
{"business": {"path": "XXXX", "host": XXXX, "is_team": true, "subscription_type": "Business"}, "personal": {"path": "XXXX", "host": XXXX, "is_team": false, "subscription_type": "Basic"}}

**Comment by gave92** [comment]
Thanks @v-aravind this is enough info for us to add support for teams accounts 👍

**Comment by yaichenbaum** [comment]
@v-aravind This feature was implemented with #3101.
